File: StudentCard.Application/Portal/Dtos/EducationBasicDto.cs
using StudentCard.Infrastructure.Helpers.Dtos;

namespace StudentCard.Application.Portal.Dtos
{
    public class EducationBasicDto
    {
        public int Id { get; set; }
        public InstitutionDto Institution { get; set; }

        public NomenclatureDto Speciality { get; set; }

        public NomenclatureDto EducationalQualification { get; set; }

        public NomenclatureDto EducationalForm { get; set; }

        public StudentStatusDto Status { get; set; }
    }
}

File: WebApplication/src/modules/portal/dtos/basic/education-basic.dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";
import { InstitutionDto } from "../helpers/institution.dto";
import { StudentStatusDto } from "../helpers/student-status.dto";

export class EducationBasicDto {
  id: number;
  institution: InstitutionDto;
  speciality: NomenclatureDto;
  educationalQualification: NomenclatureDto;
  educationalForm: NomenclatureDto;
  status: StudentStatusDto;
}